common/views/components/PageHeader/PageHeader.js

```diff
@@ -126,6 +125,25 @@ export function getHeroPicture(
   );
 }
 
+function addFreeLabel(labelsList) {
+  const freeLabel = {
+    text: 'Free',
+    labelColor: 'black',
+  };
+  if (labelsList) {
+    return {
+      ...labelsList,
+      ...{
+        labels: [freeLabel, ...labelsList.labels],
+      },
+    };
+  } else {
+    return {
+      labels: [freeLabel],
+    };
+  }
```

Comment:
```suggestion
  const freeLabel = {
    text: 'Free',
    labelColor: 'black',
  };
  const labels = (labelListProps?.labels ?? []).concat(freeLabel);
  return { ...(labelListProps ?? {}), labels }
```

This method seems quite verbose, I wondered if there was something more succinct.